docs/tutorials/tutorial-delete-data.md

```diff
@@ -165,16 +153,29 @@ A Kill Task spec has been provided at `quickstart/tutorial/deletion-kill.json`.
 curl -X 'POST' -H 'Content-Type:application/json' -d @quickstart/tutorial/deletion-kill.json http://localhost:8081/druid/indexer/v1/task
 ```
 
-After this task completes, you can see that the disabled segments have now been removed from deep storage:
+Once this task is finished, you will observe that the disabled segments, specifically segments for hours 18, 19, 22, and 23 have been successfully deleted from deep storage.
```

Comment:
The segments for hours 22-23 have been mentioned a few times already, but where did the segments for hours 18-19 get disabled?
```suggestion
When the task finishes, note that Druid deleted the disabled segments from deep storage.
```